core/src/main/java/org/apache/druid/data/input/impl/MapInputRowParser.java

```diff
@@ -50,7 +51,7 @@ public MapInputRowParser(
   {
     final List<String> dimensions = parseSpec.getDimensionsSpec().hasCustomDimensions()
                                     ? parseSpec.getDimensionsSpec().getDimensionNames()
-                                    : Lists.newArrayList(
+                                    : new ArrayList<>(
```

Comment:
Please remove the useless `import com.google.common.collect.Lists`.

Reply:
woops, this is an accidental commit I did when looking at something else, will revert